docs/02-get-started/04-trigger-workload-with-event.md

```diff
@@ -123,7 +123,7 @@ In your Function's **Configuration** tab, click on the APIRule's **Host**. This
 
 ## Create a Subscription
 
-Next, to subscribe to an event so that we can actually listen for it, we need a [Subscription](../05-technical-reference/00-custom-resources/evnt-01-subscription.md) custom resource. We're going to be listening for an event of type `order.received.v1`. 
+Next, to subscribe to an event so that we can actually listen for it, we need a [Subscription](../05-technical-reference/00-custom-resources/evnt-01-subscription.md) and an [Application](https://kyma-project.io/docs/kyma/latest/05-technical-reference/00-custom-resources/ac-01-application/#documentation-content) custom resources. We're going to be listening for an event of type `order.received.v1`. 
 
 <div tabs name="Create a Subscription" group="trigger-workload">
   <details open>
```

Comment:
One thing that's interesting is that the kubectl version works even without the application created, so I wonder if the **Application** field should filled in by selecting a value from the drop-down list, or whether it should rather be a text field to fill in (I assume it would work, as you also provide the name of the app in the **eventType.value** (line 153) in the Subscription CR via kubectl and it just works. 

Can you take it up with your team?

Reply:
As discussed via DM, an option to either type in an Application or select it from the drop-down list will be added (i.e. both will be possible), which solves the issue. Thanks!